test/Withdraw.t.sol

```diff
@@ -12,16 +12,18 @@ contract BoostWithdrawTest is BoostTest {
         _mintAndApprove(owner, depositAmount, depositAmount);
```

Comment:
Maybe we should rename it `Burn.t.sol` ?

Reply:
yep